examples/license-classifications.yml

```diff
@@ -5,374 +5,375 @@
 # To demonstrate how one can insert a custom written offer
 # include_source_code_offer_in_notice_file has been set to
 # true for all licenses of the GPL family.
-license_sets:
-- id: "copyleft"
-- id: "copyleft-limited"
-- id: "permissive"
-- id: "public-domain"
-licenses:
+categories:
+- name: "copyleft"
```

Comment:
Why rename `id` to `name`? (I find I'd better as it suggests it's unique)

Reply:
See the discussion [below](https://github.com/oss-review-toolkit/ort/pull/3327#discussion_r522837655).